R/derive_param_tte.R

```diff
@@ -60,6 +60,14 @@
 #'
 #'   A list of symbols created using `exprs()` is expected.
 #'
+#' @param check_type Check uniqueness
+#'
+#'   If `"warning"`, `"message"`, or `"error"` is specified, the specified message is issued
+#'   if the observations of the input dataset are not unique with respect to the
+#'   by variables and the order.
+#'
+#'  Default: `"none"`
```

Comment:
@ProfessorP-beep I would add an extra example to the `derive_param_tte()` documentation to show a case where sorting by order is useful, e.g. time to first serious AE and have a test case where one patient has 2 serious AEs on same date and you need to sort additionally by AESEQ.